web/cobrands/zurich/_colours.scss

```diff
@@ -26,3 +26,5 @@ $mobile_menu_tab_bg_col: #FFFFFF; // the white border and tab on mobile site
 $mobile_header_blue: #366AB6; // close match to Zurich logo_portal.jpg
 
 $header-top-border: solid 0.25em $mobile_menu_tab_bg_col;
+
+$search-help-margin: 0 -1em;
```

Comment:
Is this needed if it's not included on Zurich

Reply:
Nope, you’re right, not needed any more. Will remove.